src/views/list.php

```diff
@@ -17,6 +17,9 @@
 do_action( 'tribe_events_before_template' );
 ?>
 
+<!-- Title Bar -->
+<?php tribe_get_template_part( 'list/titlebar' ); ?>
```

Comment:
Lets use `list/title-bar` for the name of this file.

Reply:
Fixed.